packages/web-components/src/donut-chart/donut-chart.template.ts

```diff
@@ -44,13 +47,10 @@ export function donutChartTemplate<T extends DonutChart>(): ElementViewTemplate<
       ${when(
         x => !x.hideTooltip && x.tooltipProps.isVisible,
         html<T>`
-          <div
-            class="calloutContentRoot"
-            style="left: ${x => x.tooltipProps.xPos}px; top: ${x => x.tooltipProps.yPos}px"
-          >
-            <div class="calloutBlockContainer" style="border-color: ${x => x.tooltipProps.color};">
-              <div class="calloutLegendText">${x => x.tooltipProps.legend}</div>
-              <div class="calloutContentY" style="color: ${x => x.tooltipProps.color};">
+          <div class="tooltip" style="left: ${x => x.tooltipProps.xPos}px; top: ${x => x.tooltipProps.yPos}px">
```

Comment:
Use css logical property inset-inline-start instead of left

Reply:
Thanks for the suggestions!
Applied these changes in https://github.com/microsoft/fluentui/pull/33215